server/server.go

```diff
@@ -18,7 +19,6 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/spf13/viper"
 	"github.com/tylerb/graceful"
-	"golang.org/x/net/context"
```

Comment:
We will see in other PRs that the context package got updated. This was straight forward.